adapter/internal/logging/logging_constant.go

```diff
@@ -0,0 +1,322 @@
+/*
+ *  Copyright (c) 2021, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package logging
+
+import logging "github.com/wso2/apk/adapter/pkg/logging"
+
+// Error Log Internal Adapter(1100-1199) Constants
+const (
+	error1100 = 1100
+	error1101 = 1101
+	error1102 = 1102
+	error1103 = 1103
+	error1104 = 1104
+)
+
+// Error Log Internal API(1200-1299) Constants
+const (
+	error1200 = 1200
+)
+
+// Error Log Internal discovery(1400-1499) Config Constants
+const (
+	error1400 = 1400
+	error1401 = 1401
+	error1402 = 1402
+	error1403 = 1403
+	error1410 = 1410
+	error1411 = 1411
+	error1413 = 1413
+	error1414 = 1414
+)
+
+// Error Log Internal XDS(1700-1799) Config Constants
+const (
+	error1700 = 1700
+	error1701 = 1701
+	error1702 = 1702
+	error1703 = 1703
+	error1704 = 1704
+	error1705 = 1705
+	error1706 = 1706
+	error1707 = 1707
+	error1709 = 1709
+	error1710 = 1710
+	error1711 = 1711
+)
+
+// Error Log Internal intercepter(1800-1899) Config Constants
+const (
+	error1800 = 1800
+	error1801 = 1801
+)
+
+// Error Log Internal OASParser(2200-2299) Config Constants
+const (
+	error2200 = 2200
+	error2201 = 2201
+	error2204 = 2204
+	error2205 = 2205
+	error2206 = 2206
+	error2207 = 2207
+	error2208 = 2208
+	error2209 = 2209
+	error2210 = 2210
+	error2211 = 2211
+	error2212 = 2212
+	error2231 = 2231
+	error2234 = 2234
+	error2235 = 2235
+	error2236 = 2236
+	error2237 = 2237
+	error2238 = 2238
+)
+
+// Error Log Internal GRPC(2700-2799) Config Constants
+const (
+	error2700 = 2700
+)
+
+// Mapper used to keep error details for error logs
+var Mapper = map[int]logging.ErrorDetails{
+	error1100: {
+		ErrorCode: error1100,
+		Message:   "Failed to listen on port: %v, error: %v",
+		Severity:  "BLOCKER",
```

Comment:
shall we use constants for severity levels. 
```suggestion
		Severity:  BLOCKER,
```

Reply:
Fix https://github.com/wso2/apk/pull/765/commits/e54fa1f9f46385c86083062398f3215e85a08f99